src/main/drivers/pwm_output_dshot.c

```diff
@@ -101,7 +93,12 @@ FAST_CODE void pwmDshotSetDirectionOutput(
 #ifdef USE_DSHOT_TELEMETRY
     if (!output) {
         motor->isInput = true;
-        motor->timer->inputDirectionStampUs = micros();
+        if (!inputStamp) {
+            inputStamp = micros();
```

Comment:
Common practice in Betaflight is to use the suffix `...Us` for microsecond counts.